app/pb_kits/playbook/pb_nav/_item.jsx

```diff
@@ -32,21 +32,31 @@ const NavItem = ({
           onClick={onClick}
       >
         <If condition={iconLeft}>
-          <Icon
-              className="pb_nav_list_item_icon_left"
-              fixedWidth
-              icon={iconLeft}
-          />
+          <div
+              className="pb_nav_list_item_icon_section"
+              key={iconLeft}
```

Comment:
I'm trying to make sense out of using a key here. I think this will need to be a little more unique. For example, what if two different icons want to use the same icon... Their keys would be the same, and that would create a problem, right?

Reply:
I see your point. What about if we add ```key={`L-${iconLeft}`}``` and ```key={`R-${iconRight}`}``` ? I think that could solve the problem

Reply:
I think that's not a problem @jasperfurniss since React require unique key only within siblings: https://reactjs.org/docs/reconciliation.html#keys

Reply:
And the usage of the key is to force re-rendering when the icon changes.